hermes-eventbus/src/main/java/xiaofei/library/hermeseventbus/HermesEventBus.java

```diff
@@ -287,22 +298,35 @@ public void call(IMainService o) {
                 }
             });
              */
-            IMainService mainService = Hermes.getInstanceInService(service, IMainService.class);
-            mainService.register(Process.myPid(), SubService.getInstance());
-            mRemoteApis.set(mainService);
-            mState = STATE_CONNECTED;
+
+//            Log.v(TAG, "Hermes " + Service.class.getCanonicalName());
+//            Log.v(TAG, "Hermes " + service.getCanonicalName());
+            if(service.getCanonicalName().equals(Service.class.getCanonicalName())) {
```

Comment:
Thank you. It is a bug, which has also be found by another user in Issue #35, which is, however, written in Chinese.

I will fix this and merge your pull request soon. Thank you very much. And by the way, sorry for keeping you waiting for such a long time.